test/dates/io.jl

```diff
@@ -317,9 +317,9 @@ f = "ymd"
 @test Dates.DateTime(string(dt)) == dt
 
 # Vectorized
-dr = ["2000-01-01", "2000-01-02", "2000-01-03", "2000-01-04", "2000-01-05"
-     , "2000-01-06", "2000-01-07", "2000-01-08", "2000-01-09", "2000-01-10"]
-dr2 = [Dates.Date(2000):Dates.Date(2000, 1, 10);]
+dr = ["2000-01-01", "2000-01-02", "2000-01-03", "2000-01-04", "2000-01-05",
+      "2000-01-06", "2000-01-07", "2000-01-08", "2000-01-09", "2000-01-10"]
+dr2 = [Dates.Date(2000) : Dates.Date(2000, 1, 10);]
```

Comment:
when the index expressions are short yeah, but when they're both long function calls the `:` sandwiched in the middle doesn't stand out much